Makefile

```diff
@@ -23,6 +23,9 @@ eastwood:
 cljfmt:
 	lein with-profile +$(VERSION),+cljfmt cljfmt check
 
+kondo:
+	clj-kondo --lint src
```

Comment:
`test` also deserves linting

(same in https://github.com/clojure-emacs/cider-nrepl/)

Reply:
Agreed.